sktime/split/singlewindow.py

```diff
@@ -34,22 +34,58 @@
 class SingleWindowSplitter(BaseSplitter):
     r"""Single window splitter.
 
-    Split time series once into a training and test set.
-    See more details on what to expect from this splitter in :class:`BaseSplitter`.
+    Split time series into a single training and test set window.
 
-    Test window is defined by forecasting horizons
-    relative to the end of the training window.
-    It will contain as many indices
+    The training set is defined based on a "window".
+    The endpoint of the training set is determined by
+    `the size of the time series - fh[-1] - 1`,
+    and the starting point is calculated as `endpoint - window_length + 1`.
+    If the starting point is negative, it will be set to 0.
+
+    If the time points in the data are :math:`(t_0, t_1, t_2, \ldots, t_{N - 1})`,
+    the training windows will be all indices in the intervals
+
+    .. math:: [t_{N-fh[-1] - w}, \ldots, t_{N-fh[-1] - 1}]
+
+    where :math:`w` is the window length and N the size of the time series.
+
+    The test window will contain as many indices
     as there are forecasting horizons provided to the ``fh`` argument.
-    For a forecasating horizon :math:`(h_1,\ldots,h_H)`, the training window will
-    consist of the indices :math:`(k_n+h_1,\ldots,k_n+h_H)`.
+    In particularly, they will be equal to the endpoint plus the forecasting horizon.
+
+    For a forecasting horizon :math:`(h_1,\ldots,h_H)`, the test indices
+    will consist of the indices :math:`(k+h_1,\ldots,k+h_H)`,
+    where k is the end of the training window.
+
+    **Important Notes:**
```

Comment:
leave a newline, or the bullet point list will not display. Check the rendering by looking at "details" near the readthedocs CI element.